planutils/packages/htntranslate/uninstall

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+rm htntranslate-hpddl.sif
```

Comment:
Does the `wget` in the install script already get rid of the archive? Otherwise, you should clean it up here too.

Reply:
Yes, wget does not store the archive on disk. No cleanup is necessary.